ssg.js

```diff
@@ -39,16 +39,21 @@ if (config.parsedParams.help) {
  * At this point we know that we will have do do the work
  */
 
-// Verify input is given
-if (!config.parsedParams.input) {
-  // eslint-disable-next-line no-console
-  console.log('Input parameter missing');
+// Verify input or config is given
+let outputDirPath = DEFAULT_RESULT_FOLDER;
+let language = 'en-CA';
+if(config.parsedParams.input) {
+  language = config.parsedParams.language || language;
+  outputDirPath = config.parsedParams.output || outputDirPath;
+} else if(config.parsedParams.config) {
+  config = parseConfigArgs(config.parsedParams.config)
+  language = config.parsedParams.language || language;
+  outputDirPath = config.parsedParams.output || outputDirPath;
+} else {
+  console.log('Input or config parameter missing');
   process.exit(1);
 }
 
-const { language = 'en-CA', output: outputDirPath = DEFAULT_RESULT_FOLDER } =
-  config.parsedParams;
-
```

Comment:
This way we'll have to manually maintain each parameter here later.

```suggestion
let { parsedParams } = config;
if (parsedParams.config) {
  config = {
    ...config,
    ...parseConfigArgs(config.parsedParams.config).parsedParams
  };
}

/ Verify input is given
if (!config.parsedParams.input) {
  // eslint-disable-next-line no-console
  console.log('Input parameter missing');
  process.exit(1);
}

const { language = 'en-CA', output: outputDirPath = DEFAULT_RESULT_FOLDER } =
  config;
```

This should automatically merge everything together later on